docs/_posts/2019-09-05-ch-ch-ch-ch-changelog.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Ch-ch-ch-ch-changelog!
+---
+
+It's been two weeks since the last Bolt update and what a two weeks it's been.
+
+In that time, we've completed our move from Jira to GitHub for tracking Bolt issues and work-in-progress. If you want to follow what we're working on, you can now simply [consult our project board](https://github.com/puppetlabs/bolt/projects/2).
+
+We also just released Bolt 1.30 (**thirty**!). You can check out the full list of changes in our shiny new [CHANGELOG.md file](https://github.com/puppetlabs/bolt/blob/master/CHANGELOG.md).
+
+This release has some helpful improvements to output and other fixes, but the highlight is definitely the new pluggable `puppet_library` hook. This satisfies an extremely popular request to be able to customize how Bolt installs Puppet on systems when you call `apply_prep()`.
+
+By default, Bolt will use the `puppet_agent::install` task to install the very latest version of Puppet. That's great if all you care about is getting Puppet on the system so you can use `apply()`. But if you're using Bolt alongside an established Puppet deployment, you probably care a bit more about which version of Puppet gets installed and where it's downloaded from. You may also want to do some additional configuration. With the new `puppet_library` hook, you can do all that and more!
+
+For instance, you can use the `puppetlabs-bootstrap` module to install a Puppet Enterprise agent, connected to a specific puppet master.
+
+```yaml
+# bolt.yaml
+plugin_hooks:
+  puppet_library:
+    plugin: task
+    task: bootstrap
+    params:
```

Comment:
Where did this get copied from? I ask because it needs to be `parameters` or parameters are silently ignored. I thought I fixed it everywhere with https://github.com/puppetlabs/bolt/pull/1197

Reply:
It came from pre-docs and it looks like the current version of the code I copied is correct. Maybe I had an old version checked out when I copied it.